cinema/urls.py

```diff
@@ -7,6 +7,7 @@
     CinemaHallViewSet,
     MovieViewSet,
     MovieSessionViewSet,
+    OrderViewSet
```

Comment:
Code Style: The imports should be grouped using parentheses `()` as per the checklist. Consider changing the import statement to use parentheses for better readability.